### DISCUSSION TITLE
New difficulty not working. What in particular is wrong?

### DISCUSSION THREAD

**Discussion by daisukidaisy** [Leather128/LeathersFunkinEngine#47] [Q&A] [answered]
I am trying to add a 4th difficulty for the base game songs (in the vanilla folders). I have charts for Week 1 and have added the 4th difficulty for those specific songs in freeplaySonglist.txt.
Unfortunately, I can't access the charts for the 4th difficulty in Freeplay, whereas the vanilla difficulties still work just fine.
I haven't yet enabled the 4th difficulty for Story mode either. This might be the problem, but that's just a hunch, so I'd rather get a solid answer from someone who knows what they're doing before I start screwing up things that might make the game unstable.
I don't want to be forced to duplicate the vanilla songs in the mods folder.

**Comment by Leather128** [comment, marked as answer]
you cannot add extra difficulties into the base files, it has to be in a mod or else the game cannot recognize it. don't worry about duplication btw, you just have to add the extra chart into the mod (and also an entry in freeplaySonglist.txt for the new difficulty if you want the mod to work for people who haven't put it in their own freeplaySonglist).
